plasmapy/formulary/collisions.py

```diff
@@ -2214,34 +2305,51 @@ def __init__(
         self.test_particle = test_particle
         self.field_particle = field_particle
         self.v_a = v_a
-        self.T_a = (
-            T_a
-            if T_a is not None
-            else (0.5 * test_particle.mass * v_a**2 / k_B).to(u.K)
-        )
-        self.n_b = n_b
+        self.T_a = T_a
+        self.n_a = n_a
         self.T_b = T_b
+        self.n_b = n_b
         self.Coulomb_log = (
             Coulomb_log
             if isinstance(Coulomb_log, u.Quantity)
             else Coulomb_log * u.dimensionless_unscaled
         )
 
+    @cached_property
+    def _v_T_a(self):
+        """
+        The thermal velocity of the test particle species.
+        """
+
+        return (
+            self.v_a
+            if self.v_a is not None
+            else thermal_speed(self.T_a, self.test_particle)
+        )
+
     @cached_property
     def _mass_ratio(self):
         return self.test_particle.mass / self.field_particle.mass
 
     @cached_property
+    @_validate_arguments(expected_arguments=["n_b", ("T_a", "v_a")])
     def momentum_loss(self):
         """
         The momentum loss rate due to collisions.
+
+        In order to access this attribute, ``n_b`` must be specified in addition
+        to either ``T_a`` or ``v_a``.
```

Comment:
These are all enforced by your validate_arguments decorator, right? Should all be tested too.